test/default-resolver.js

```diff
@@ -0,0 +1,3 @@
+export default function defaultResolver(path) {
```

Comment:
Are we using this definition of defaultResolver anywhere? I see us importing only `src/default-resolver` elsewhere.

Reply:
yes. It's used extensively in `client.js`